src/controller/data_model/controller-clusters.matter

```diff
@@ -2827,6 +2827,10 @@ client cluster IcdManagement = 70 {
     optional octet_string<16> verificationKey = 1;
   }
 
+  response struct StayActiveResponse = 4 {
```

Comment:
this does not show up as a parameter to `StayActiveRequest`. Maybe a `response=` is missing in the XML?

Reply:
went a bit too fast. updated